### PR TITLE
update finding details page screenshot and viewing finding activity i…

### PR DESCRIPTION
**Pull request by sabrinabrogren**
…nstructions

fixes APP-3219

# Thanks for improving Semgrep Docs 😀

image

### Please ensure

- [X] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [X] This change has no security implications or else you have pinged the security team
- [X] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-app/findings.md [hunk 1]
```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
 
 <RemoveRule />
 
-#### Viewing and adding comments to finding's history
+#### Viewing and adding comments to finding's activity
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:208]: We are now adding notes, not comments as it seems: Screenshot 2023-01-31 at 21 51 47

```suggestion
#### Viewing and adding notes to findings
```

File: docs/semgrep-app/findings.md [hunk 2]
```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
...
 
-To **view and add comments** to history of a finding:
+To **view and add comments** to activity history of a finding:
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:210]:
```suggestion
To **view and add notes** to the activity of a finding:
```

File: docs/semgrep-app/findings.md [hunk 3]
```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
...
 
-1. Click the <i className="fa-regular fa-arrow-up-right-from-square inline_svg"></i> **Finding details** icon next to the finding’s status.
-2. Find the comments in the **History** section.
+1. Click the heading of the finding card.
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:212]:
```suggestion
1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the heading of the finding card.
```

File: docs/semgrep-app/findings.md [hunk 4]
```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
...
+2. Find the comments in the **Activity** section.
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:213]:
```suggestion
2. View, or add the notes in the **Activity** section. To add a new note, click plus **New note**.
```

Comment (adamkvitek) [inline docs/semgrep-app/findings.md:213]: @sabrinabrogren where can I find the new plus icon in the circles? I cannot find it in FontAwesome. Please, if you have a link to this icon and we can reuse it in the docs, let us know. :-) Screenshot 2023-01-31 at 21 58 12

**Review by adamkvitek — COMMENTED** [review]
@sabrinabrogren left a few suggestions, please check. Thank you for this update! 🚂

**Review by adamkvitek — APPROVED** [review]

File: docs/semgrep-app/findings.md (hunk 4 above)
Comment (sabrinabrogren) [reply docs/semgrep-app/findings.md:213]: Ah, yeah that's actually combination of an icon and the [mantine timeline component](https://mantine.dev/core/timeline/). So the blue circle is from the component, and the plus in the middle is the actual icon. Should I still add the icon to the docs or would that be confusing?

Comment (adamkvitek) [reply docs/semgrep-app/findings.md]: Please, if we can somehow add this icon, it will be great. @sabrinabrogren

**Pull request merged**